pkgs/tools/misc/nitch/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, nimPackages, fetchFromGitHub }:
+nimPackages.buildNimPackage rec {
+  pname = "nitch";
+  version = "0.1.6";
+  nimBinOnly = true;
+  src = fetchFromGitHub {
+    owner = "quasigod-io";
+    repo = "nitch";
+    rev = "42ad6899931dd5e0cec7b021c2b7e383fcc891f3";
```

Comment:
I see you're the maintainer. Would you like to tag the release, so it can be referred here by the tag name instead of a bare hash? It makes future updates simpler.

Reply:
I am currently maintaining a fork as the original is not currently being updated. My fork adds support for NixOS specific outputs. Would it be better to package the original as well as a fork, or package it using my fork and switch to the original if it becomes updated?

Reply:
If that's the case, what you could do is package the original and pull the nixos-specific patch separately similar to (for example) this:
https://github.com/NixOS/nixpkgs/blob/d3270d6b32c6a5404c2429e20ab75969d8376d1d/pkgs/tools/virtualization/udocker/default.nix#L28-L33

You can reference your upstream PR by taking the github commit link and appending patch to it - in your case `https://github.com/unxsh/nitch/commit/6831cf96144f58c4da298a0bc9b50d33056f6c08.patch`

This way it will be easy to drop the patch when the version is bumped.

Reply:
Got it, will do that, thank you!

Reply:
In this case, the package has had many significant commits since the last release. Should I still use the last release tag or the latest commit?